.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,30 @@
+# Run ShellCheck on PRs and master
+
+# For more context, see:
+# https://github.com/openedx/.github/blob/master/docs/decisions/0001-shellcheck.rst
+
+name: ShellCheck
+
+on:
+  pull_request:
+  push:
+    branches:
+    - master
+
+permissions:
+  contents: read
+
+jobs:
+  shellcheck:
+    strategy:
+      matrix:
+        os: ["ubuntu", "macos"]
+    #uses: openedx/.github/.github/workflows/shellcheck.yml@master
+    uses: kdmccormick/openedx.github/.github/workflows/shellcheck.yml@kdmccormick/shellcheck-impl
```

Comment:
Just making a note that this needs to be updated before we merge this, I see that you have the comment ready to swap, when the `.github` PR has merged.